joomla/joomla-cms
# Change type of field "value" in table #_fields_values from text to mediumtext

Change type of field "value" in table `#_fields_values` from `text` to `mediumtext` for MySQL ONLY. PostgreSQL have only text field type with unlimited length. Pull Request for Issue #36065 and remake a PR #42605 for Joomla 5.1.0

# Testing instructions
1. Go to bottom of this PR and click **Show all checks** in **All checks have passed**  section
2. Click on **Details** in **download** line
![image](https://github.com/joomla/joomla-cms/assets/6236403/a6ebdd4f-84a9-4255-a98a-3fb859d0bfd9)
3. You will go to https://artifacts.joomla.org/drone/joomla/joomla-cms/5.1-dev/42606/downloads/72696/
4. Download Full installation package and update package of Joomla 5.1 + this PR
![image](https://github.com/joomla/joomla-cms/assets/6236403/9be6d5c6-472a-45bb-a367-905c73a323e7)
5.  Make a clean install from `Joomla_5.1.0-alpha3-dev+pr.42606-Development-Full_Package.zip`
6. Go to PhpMyAdmin, select database you used for clean install and go to the `#__field_values` table. Then go to **structure** section and check that type of `value `field is `mediumtext`.
![image](https://github.com/joomla/joomla-cms/assets/6236403/639efc88-6317-47dd-8bd6-a1b4ccc38631)
7. Clean your database and test domain folder and install a Joomla 5.0.1 
8. Check `value` field type in PHP My Admin in `#__field_values` table. It is a `text` type now.
![image](https://github.com/joomla/joomla-cms/assets/6236403/29b3f9e0-6b95-409e-9448-6ad560757c9c)
9. Then go to **admin panel / System / Update / Joomla** . Press the **Upload and Update** button
![image](https://github.com/joomla/joomla-cms/assets/6236403/45abfbfa-2457-4618-8369-5fbe099352d2)
10. Upload  the `Joomla_5.1.0-alpha3-dev+pr.42606-Development-Update_Package` and update
![image](https://github.com/joomla/joomla-cms/assets/6236403/8e9390c2-35e7-4b1e-811b-55e2e7016260)
11. Then go to PhpMyAdmin again and check the `#__field_values` table - type of field `value`. It could be a `mediumtext`
![image](https://github.com/joomla/joomla-cms/assets/6236403/38b85f0d-e381-48a4-b12c-bbe6c44bc45f)
